cmd/lakectl/cmd/commit.go

```diff
@@ -9,30 +9,41 @@ import (
 	"github.com/treeverse/lakefs/pkg/uri"
 )
 
-var commitCreateTemplate = `Commit for branch "{{.Branch.Ref}}" completed.
+var (
+	errInvalidKeyValueFormat    = fmt.Errorf("invalid key/value pair - should be separated by \"=\"")
```

Comment:
nit: separate errors to another var block.

Reply:
also, you can use back quote (`) so you don't need to escape \" (suggestion)